packages/builder/builder/src/plugins/tsChecker.ts

```diff
@@ -73,6 +73,14 @@ export const builderPluginTsChecker = (): BuilderPlugin => {
             ? config.tools.tsChecker
             : {},
         );
+        if (
+          api.context.bundlerType === 'rspack' &&
+          chain.get('mode') === 'production'
+        ) {
+          logger.warn(
```

Comment:
logger.warn` is too negative, we can use `logger.info`, `.

```suggestion
          logger.info(
```